src/lj_arch.h

```diff
@@ -445,6 +445,16 @@
 #error "Need at least GCC 4.8 or newer"
 #endif
 #endif
+#elif LUAJIT_ARCH_PPC
+#if __clang__
+#if ((__clang_major__ < 3) || ((__clang_major__ == 3) && __clang_minor__ < 5)) && !defined(__NX_TOOLCHAIN_MAJOR__)
+#error "Need at least Clang 3.5 or newer"
```

Comment:
The check should fail with the version numbers you mentioned in the comment, i.e. 3.4.  That said, this is clearly not clang 8.0 as you had mentioned in #55 .